packages/cli-kit/src/public/node/fs.ts

```diff
@@ -100,16 +100,14 @@ export async function readFile(path: string, options: ReadOptions = {encoding: '
 }
 
 /**
- * Synchronously reads a file and returns its content as a string.
+ * Synchronously reads a file and returns its content as a buffer.
  *
  * @param path - Path to the file to read.
- * @param options - Options to read the file with (defaults to utf-8 encoding).
  * @returns The content of the file.
  */
-export function readSync(path: string, options: object = {encoding: 'utf-8'}): string {
+export function readSync(path: string): Buffer {
```

Comment:
could you take the opportunity to rename it to `readFileSync`? I forgot to do it and is not following the same name pattern as other functions